src/models/local.js

```diff
@@ -0,0 +1,56 @@
+import OpenAIApi from 'openai';
+
+export class local {
+    constructor(model_name) {
+        this.model_name = model_name;
+
+        this.openai = new OpenAIApi(
+            {
+                apiKey: process.env.OPENAI_API_KEY,
+                baseURL: process.env.OPENAI_API_BASE,
```

Comment:
No need to create a new file. Instead let's add the baseURL to the openAIConfig in the GPT model file. Then have the model default to GPT instead of local when it doesn't recognize the name.